### DISCUSSION TITLE
Help Me

### DISCUSSION THREAD

**Discussion by TrendSpider001** [community/community#57192] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi github team.
I can't upload avatar on my profile.
Please help me.
Thanks

**Comment by danielalyoshin** [comment]
Please provide some more information. Are you getting an error message when trying to upload the image?

**Reply by TrendSpider001** [reply]
yes, of course.
![Screenshot_2](https://github.com/community/community/assets/127406489/9cb79667-ac4b-48d7-8ecd-1e3857c71d5e)
Please help me

**Reply by danielalyoshin** [reply]
As stated in the GitHub documentation [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#changing-your-profile-picture):

> Your profile picture should be a PNG, JPG, or GIF file, and it must be less than 1 MB in size and smaller than 3000 by 3000 pixels. For the best quality rendering, we recommend keeping the image at about 500 by 500 pixels.

Does the file you are uploading fulfill these requirements?

**Reply by TrendSpider001** [reply]
Thanks for your answer.
But I can't upload yet.
![1](https://github.com/community/community/assets/127406489/f2c6e966-b8eb-4c81-bede-ac2634f9f3fb)
Excuse me sir, please help me.

**Reply by danielalyoshin** [reply]
Does the issue persist even when trying other images, or is it unique to that image in particular?

Try uploading a different image and see if you get the same error.

**Reply by TrendSpider001** [reply]
Issue persist even when trying other images.
Please help me.

**Reply by TrendSpider001** [reply]
and Can you share me your contact information?
Skype or telegram.

**Reply by danielalyoshin** [reply]
I use neither Skype nor Telegram, but I can direct you to #48077, which deals with your issue. Perhaps one of the solutions there will be of help.

**Comment by TrendSpider001** [comment]
I have done!
